### ISSUE TITLE
Impaired renal tubular reabsorption of bicarbonate

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6868]
**Preferred term label:**
Impaired renal tubular reabsorption of bicarbonate

**Synonyms**

**Definition (free text, please give PubMed ID)**
Decreased renal tubular reabsorption of bicarbonate.
PMID: 31474092

**Comment**
A fractional bicarbonate excretion of ≥ 15% is considered pathologic, when the plasma bicarbonate is normal.

**Parent term (use hpo.jax.org/app)**
Abnormality of renal resorption HP:0011038

**Associated terms** 
Renal Fanconi syndrome HP:0001994, 
Proximal renal tubular acidosis HPO: 0002049

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
added
